setup.py

```diff
@@ -31,32 +31,19 @@ def package_files(directory):
     packages=find_packages(exclude=["tests", "tests.*"]),
     package_data={"mlflow": js_files + models_container_server_files + alembic_files},
     install_requires=[
-        "alembic<=1.4.1",
-        # Required
-        "azure-storage-blob",
```

Comment:
Artifact repositories should also be lazy loaded based off of dependencies https://github.com/mlflow/mlflow/blob/master/mlflow/store/artifact/artifact_repository_registry.py